catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/pom.xml

```diff
@@ -227,47 +241,22 @@
                             spatial-wfs-v1_1_0-common,
                             spatial-wfs-source,
                             spatial-ogc-common,
-                            spatial-wfs-converter;scope=compile|runtime,
-                            spatial-wfs-v1_1_0-converter,
                             catalog-core-api-impl,
-                            ddf-security-common,
                             platform-util,
-                            platform-util-unavailableurls,
-                            geospatial,
-                            spatial-wfs-v1_0_0-schema-bindings,
-                            security-core-impl
+                            geospatial
                         </Embed-Dependency>
+                        <Private-Package>
+                            org.codice.ddf.spatial.ogc.wfs.catalog.mapper.impl
+                        </Private-Package>
                         <Import-Package>
                             !org.bouncycastle.crypto.*,
                             net.opengis.filter.v_1_1_0;version="[${org.jvnet.ogc.wfs.1.1.0.version},${org.jvnet.ogc.wfs.1.1.0.version}]",
                             *
                         </Import-Package>
-                        <Embed-Transitive>true</Embed-Transitive>
                         <Include-Resource>{maven-resources},target/classes/describable.properties</Include-Resource>
                     </instructions>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-enforcer-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>check-artifact-size</id>
-                        <phase>verify</phase>
-                        <goals>
-                            <goal>enforce</goal>
-                        </goals>
-                        <configuration>
-                            <rules>
-                                <ArtifactSizeEnforcerRule
-                                    implementation="org.codice.maven.artifactsize.ArtifactSizeEnforcerRule">
-                                    <maxArtifactSize>1.3_MB</maxArtifactSize>
```

Comment:
Removed this because the artifact is now under the default maximum size of 1 MB.